packages/stark-ui/src/modules/date-picker/components/date-format.constants.ts

```diff
@@ -1,10 +1,11 @@
+import { MatDateFormats } from "@angular/material/core";
 /**
  * Object containing the format of the dates displayed in the date picker
  * All formats can be found here: https://momentjs.com/docs/#/displaying/
  */
-export const STARK_DATE_FORMATS: Object = {
+export const STARK_DATE_FORMATS: MatDateFormats = {
```

Comment:
Can you expand the JSDoc of this constant to explain that we use Moment dates internally? So it is clear why we expect one of the moment formats from https://momentjs.com/docs/#/displaying/